flink-runtime/src/main/java/org/apache/flink/runtime/security/token/DelegationTokenUpdater.java

```diff
@@ -38,18 +38,18 @@ public static void addCurrentUserCredentials(byte[] credentialsBytes) throws IOE
             throw new IllegalArgumentException("Illegal credentials tried to be set");
         }
         Credentials credentials = DelegationTokenConverter.deserialize(credentialsBytes);
-        LOG.debug("Updating delegation tokens for current user");
+        LOG.info("Updating delegation tokens for current user");
```

Comment:
@gaborgsomogyi are you changing the log level because you reconsidered the usefulness of these messages or just to help with testing? If the latter is true we should instead set the log config to be debug level in the e2e test.

Reply:
My main intention is to see token usage in info level. These logs are produced small amount of logs (maybe 10-20 line per day) but they're essential to see the tkoen framework lifecycle. As a side effect I'm using one of them as assertion criteria.